lib/elastic_apm/graphql.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module ElasticAPM
+  # @api private
+  module GraphQL
+    TYPE = 'app'
+    SUBTYPE = 'graphql'
+
+    PREFIX = 'GraphQL: '
+    UNNAMED = '[unnamed]'
+    MORE_THAN_FIVE = '[more-than-five-queries]'
+    CONCATENATOR = '+'
+
+    KEYS_TO_NAME = {
+      'lex' => 'graphql.lex',
+      'parse' => 'graphql.parse',
+      'validate' => 'graphql.validate',
+      'analyze_multiplex' => 'graphql.analyze_multiplex',
+      'analyze_query' => 'graphql.analyze_query',
+      'execute_multiplex' => 'graphql.execute_multiplex',
+      'execute_query' => 'graphql.execute_query',
+      'execute_query_lazy' => 'graphql.execute_query_lazy',
+      # "execute_field" => "graphql.execute_field",
+      # "execute_field_lazy" => "graphql.execute_field_lazy",
+      # "authorized" => "graphql.authorized",
```

Comment:
Why are these commented out?

Reply:
They are available but left out deliberately. I thought it would be nice to include them in this form to 1) show what we skip and 2) be easy to find if we want to stop skipping them.

Reply:
Are all of these names common across all GraphQL implementations in different languages? Maybe it's worth noting in the spec which ones we include and exclude just so there's some documentation somewhere.

Reply:
I don't think they are necessarily the same, no. There's already a note (sort of) about this: https://github.com/elastic/apm/pull/208/files#diff-9c3af4b12c4c30fd39c2bcf34e7bc375R146

Reply:
Ok, so perhaps the spec isn't the right place to document which are included and excluded. But I still don't know myself why they are commented out - would you mind explaining here or putting a comment in the source?